prefs/Resources/fr.lproj/Common.strings

```diff
@@ -18,7 +18,7 @@
 
 	<!-- Heading above the "Email Support" button. -->
 	<key>EMAIL_SUPPORT_HEADER</key>
-	<string>Des problèmes?</string>
+	<string>Des problèmes ?</string>
```

Comment:
For french (France) we need a space, but for french (Canada) there is no space.
Here is a _long_ discussion about that on [wordreference](http://forum.wordreference.com/threads/fr-space-before-a-colon-semicolon-question-mark-or-exclamation-point.54376/?hl=fr).